packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -93,14 +120,23 @@ export default {
 
             this.updateMessage();
 
-            if (modalContext) modalContext.close();
+            if (modalContext) {
+                modalContext.close();
+            }
+        },
+
+        showOrderHistory () {
+            window.location.assign('order-history');
+
+            this.updateMessage();
         }
     }
 };
+
 </script>
 
 <style lang="scss" module>
-.c-checkout-redirectButton {
-    margin: spacing(x4) 0 spacing(x0.5);
+.c-checkout-errorDialogButton {
+    margin: spacing(x4) spacing(x1.5) spacing(x0.5);
```

Comment:
Will these styles be ok when there's only one button? i.e if there's no duplicate order because now a single button will have spacing to the left and right?

Reply:
Yes looks fine